packages/block-library/src/post-time-to-read/variations.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { clock } from '@wordpress/icons';
```

Comment:
```suggestion

/**
 * Internal dependencies
 */
import icon from './icon';
```

The `clock` icon exists locally, not in the icons package.